jenkins/e2e-image/e2e-runner.sh

```diff
@@ -176,6 +176,11 @@ function install_google_cloud_sdk_tarball() {
     mkdir -p "${install_dir}"
     tar xzf "${tarball}" -C "${install_dir}"
 
+    if running_in_docker; then
+        # Delete any previously installed SDK to prevent weirdly overlapping
+        # installs from fighting each other.
+        rm -rf "${install_dir}/google-cloud-sdk"
+    fi
```

Comment:
That will leave us with stale versions of the SDK. We can use "gcloud components update", but that's not any faster than the normal path of downloading the bundle.

I can leave a TODO to resolve this properly in the future.